src/index.ts

```diff
@@ -605,6 +605,7 @@ class NixInstallerAction extends DetSysAction {
   }
 
   async install(): Promise<void> {
+    actionsCore.startGroup("Detecting Nix");
```

Comment:
Let's leave this out. Closing this group in this code path is going to be fairly difficult, and I'm not sure it'll add anything to the user experience.

```suggestion
```